### ISSUE TITLE
client.files.save throws an error

### ISSUE THREAD

**Issue by TheChrisVela** [Vonage/vonage-ruby-sdk#102]
Calling this method

```ruby
RECORDING_URL = 'https://api.nexmo.com/v1/files/aaaaaaaa-bbbb-cccc-dddd-0123456789ab'
client.files.save(RECORDING_URL, 'recording.mp3')
```
throws the following error:
```sh
/Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/2.4.0/net/http/response.rb:198:in `read_body': Net::HTTPOK#read_body called twice (IOError)
	from /Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nexmo-5.0.0/lib/nexmo/files.rb:12:in `block (2 levels) in save'
	from /Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nexmo-5.0.0/lib/nexmo/files.rb:11:in `open'
	from /Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nexmo-5.0.0/lib/nexmo/files.rb:11:in `block in save'
	from /Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nexmo-5.0.0/lib/nexmo/namespace.rb:77:in `parse_success'
	from /Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nexmo-5.0.0/lib/nexmo/namespace.rb:61:in `parse'
	from /Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nexmo-5.0.0/lib/nexmo/namespace.rb:53:in `request'
	from /Users/chrisguzman/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nexmo-5.0.0/lib/nexmo/files.rb:10:in `save'
```

**Comment by timcraft** [comment]
@ChrisGuzman Thanks for the heads up, I'll take a look.

**Comment by timcraft** [comment]
I can reproduce, the tests aren't failing because WebMock overrides / removes the Net::HTTP behaviour which causes it to fail. I'll include a fix in the next release.

**Comment by timcraft** [comment]
Fixed in 77ecfe2 (available in v5.3.0)

**Comment by TheChrisVela** [comment]
Thanks!
